ci/keystone/keystone.sh

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env bash
+#
+# Helper script for setting up and running keystone. Most of this is taken from:
+#
+#   http://docs.openstack.org/developer/keystone/developing.html
+
+# tunable variables:
+
+# So we can avoid repeatedly downloading the repo when developing this script:
+keystone_repo=${keystone_repo:-https://git.openstack.org/openstack/keystone}
+
+# git commit to use. Travis uses this to test against different releases of
+# keystone:
+keystone_commit=${keystone_commit:-master}
```

Comment:
Good spot; I meant to bring this up and forgot. There's also the option of testing multiple versions by adding stuff to Travis's env matrix, though that will add time. I think latest stable at a minimum is a good idea. Travis also let's you tag parts of the matrix as failures acceptable, so we could keep it testing master too, and get early warning if an upcoming version is going to break something.